test/stdlib/NSSlowTaggedLocalizedString.swift

```diff
@@ -14,6 +14,8 @@ import StdlibUnittest
 
 let longTaggedTests = TestSuite("NonContiguousTaggedStrings")
 var constant = "Send Message to different Team"
+//doesn't fit in a tagged pointer because of ', but does fit in a SmallString
+var shortNonTagged = "Don't Save"
```

Comment:
Is this the "smart" apostrophe, or just `'` (ASCII: 0x27) ?

Reply:
Looks to me like it's U+0027, should we be using something like U+2019 instead so that it's not an ASCII string so that decoding as ASCII will fail?

Reply:
argh, copy-paste from the bug report has betrayed me! I'll fix that, thanks for catching it